providers/transmit_provider.ts

```diff
@@ -51,11 +51,11 @@ export default class TransmitProvider {
       return ctx.response.noContent()
     })
 
-    router.post('__transmit/unsubscribe', (ctx) => {
+    router.post('__transmit/unsubscribe', async (ctx) => {
       const uid = ctx.request.input('uid')
       const channel = ctx.request.input('channel')
 
-      const success = transmit.$unsubscribeFromChannel(uid, channel, ctx)
+      const success = await transmit.$unsubscribeFromChannel(uid, channel, ctx)
```

Comment:
No need to have an `async` function and `await` here. `$unsubscribeFromChannel` is sync.